README.md

```diff
@@ -53,6 +53,24 @@ Add it to your configuration:
 echo "[secret key]" > config/credentials/development.key
 ```
 
+### Add efile resources locally
+
+In development, we need to download the IRS e-file schemas zip manually from S3.
+
+> ℹ️ We avoid storing them in the repo because the IRS asked us nicely to try to limit distribution.
+
+- Go to [Google Docs and download this file](https://drive.google.com/drive/u/0/folders/1ssEXuz5WDrlr9Ng7Ukp6duSksNJtRATa) (ask a teammate if you don't have access)
```

Comment:
This doesn't state which file is needed. Since the folder has approx 5 files in it, you might want to either state it here, _or_ say that the user should run `rake setup:unzip_efile_schemas` in order to find out the specific filename.

Reply:
Added the file name in the docs.